### PR TITLE
Update net.bartkessels.getit.json

### PR DESCRIPTION
**Pull request by ghost**
- Update runtime version
- Add missing libsoup-2.4

### CODE REVIEW

**Comment by ghost** [comment]
@bbhtt FYI

**Comment by bbhtt** [comment]
I don't maintain this one, I just did a few runtime updates because the maintainer was MIA. If you can confirm it works, I'll merge it.

File: net.bartkessels.getit.json [hunk 1]
```diff
@@ -33,6 +33,20 @@
...
+				"-Dtests=false"
+			],
+			"sources": [
+				{
+					"type": "archive",
+					"url": "https://download.gnome.org/sources/libsoup/2.74/libsoup-2.74.3.tar.xz",
+					"sha256": "e4b77c41cfc4c8c5a035fcdc320c7bc6cfb75ef7c5a034153df1413fa1d92f13"
+				}
+			]
+		},
```
Comment (bbhtt) [inline net.bartkessels.getit.json]: Please use this from shared modules https://github.com/flathub/shared-modules/tree/master/libsoup

**Comment by ghost** [comment]
- Local test: working
- Waiting for: flathub version

resim

**Comment by bbhtt** [comment]
Shared-modules wasn't properly added see the readme https://github.com/flathub/shared-modules you are missing .gitmodules

**Comment by ghost** [comment]
> I don't maintain this one, I just did a few runtime updates because the maintainer was MIA. If you can confirm it works, I'll merge it.

It works. Could you merge it when you have time. Thanks for everything.

**Pull request merged**